cmd/pool/account.go

```diff
@@ -173,27 +207,36 @@ func newAccount(ctx *cli.Context) error {
 }
 
 func printAccountFees(client poolrpc.TraderClient, amt btcutil.Amount,
-	confTarget uint32) error {
+	satPerVByte uint64, confTarget uint32) error {
 
-	req := &poolrpc.QuoteAccountRequest{
-		AccountValue: uint64(amt),
-		Fees: &poolrpc.QuoteAccountRequest_ConfTarget{
-			ConfTarget: confTarget,
-		},
+	if confTarget == 0 {
+		fmt.Println("-- Account Funding Details --")
+		fmt.Printf("Amount: %v\n", amt)
+		fmt.Printf("Fee rate (estimated): %d sat/vByte\n", satPerVByte)
+
+		return nil
 	}
-	resp, err := client.QuoteAccount(context.Background(), req)
+
+	resp, err := client.QuoteAccount(
+		context.Background(),
+		&poolrpc.QuoteAccountRequest{
+			AccountValue: uint64(amt),
+			Fees: &poolrpc.QuoteAccountRequest_ConfTarget{
+				ConfTarget: confTarget,
+			},
+		},
+	)
 	if err != nil {
 		return fmt.Errorf("unable to estimate on-chain fees: "+
 			"%v", err)
 	}
 
 	feeRate := chainfee.SatPerKWeight(resp.MinerFeeRateSatPerKw)
-	satPerVByte := float64(feeRate.FeePerKVByte()) / 1000
-
+	feePerKVByte := float64(feeRate.FeePerKVByte()) / 1000
```

Comment:
nit: we still divide by 1000 so it's `feePerVByte`. Meaning, we can just overwrite the value of the existing variable here.